docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.9"
```

Comment:
I am not sure this should be in the root as it can not be used to start our programs. Maybe this is better suited for documentation or should be renamed to "example" or something?

The images do not exist in docker hub and your compose file will therefore fail. Doesn't block the merge for me but I'll probably change it later on.